src/cmd/serve.rs

```diff
@@ -147,31 +149,32 @@ fn handle_directory<'a, 'b>(dir: &'a fs::Directory, req: &'b HttpRequest) -> io:
     fs::NamedFile::open(path)?.respond_to(req)
 }
 
+const OPTIONAL_WATCH_DIRS: [&str; 4] = ["content", "sass", "static", "templates"];
```

Comment:
`content` is not optional, there is nothing to do for gutenberg if it isn't there